opencti-platform/opencti-front/src/private/components/settings/users/UserCreation.jsx

```diff
@@ -264,6 +258,28 @@ class UserCreation extends Component {
                     label="Organizations"
                     style={fieldSpacingContainerStyle}
                   />
+                  <Field
+                    component={SelectField}
+                    variant="standard"
+                    name="account_status"
+                    label={t('Account Status')}
+                    fullWidth={true}
+                    containerstyle={fieldSpacingContainerStyle}
+                    >
+                    {settings.platform_user_statuses.map((s) => {
+                      return <MenuItem key={s.status} value={s.status}>{s.status}</MenuItem>;
```

Comment:
What about translating account statuses?

Reply:
@Archidoit - give me a little bit - I will push a commit with the translations fixed - sorry.

Reply:
@Archidoit - Corrected.